installers/custom_ipxe/main.go

```diff
@@ -1,24 +1,96 @@
 package custom_ipxe
 
 import (
+	"encoding/json"
 	"strings"
 
+	"github.com/packethost/pkg/log"
+	"github.com/tinkerbell/boots/installers"
 	"github.com/tinkerbell/boots/ipxe"
 	"github.com/tinkerbell/boots/job"
 )
 
 func init() {
-	job.RegisterSlug("custom_ipxe", bootScript)
+	job.RegisterInstaller("ipxe", ipxeScript)
+	job.RegisterSlug("custom_ipxe", ipxeScript)
 }
 
-func bootScript(j job.Job, s *ipxe.Script) {
+func ipxeScript(j job.Job, s *ipxe.Script) {
+	logger := installers.Logger("ipxe")
```

Comment:
this won't be accurate when called with the `custom_ipxe` slug. anything that can be done to accommodate both? maybe `job.RegisterInstaller("custom_ipxe", ipxeScript)`?

Reply:
I can just change it to `custom_ipxe` since that's the package the logs will be coming from.